awx/main/management/commands/custom_venvs.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2021 Ansible, Inc.
+# All Rights Reserved
+
+from awx.main.utils.common import get_custom_venv_choices, get_custom_venv_pip_freeze
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Returns either a list of custom venv paths or outputs the pip freeze from the path passed in the argument"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
```

Comment:
I feel great about that

Reply:
I feel less great now, the way that parser works requires the flags for the variable to be optional, so it would have to be separated into 2 different commands if I want it to be just positional

Reply:
for future reference it is possible, I was just doing it wrong :)